dmutils/content_loader.py

```diff
@@ -416,6 +418,27 @@ def get_question(self, framework_slug, question_set, question):
 
         return self._questions[framework_slug][question_set][question].copy()
 
+    def get_message(
+        self, framework_slug, block, framework_status, supplier_status=None
+    ):
+
+        framework = self._content[framework_slug]
+
+        if 'messages' not in framework:
```

Comment:
I feel like I'm missing some context here.
As far as I understand `_content[framework_slug]` is a dictionary of manifest files (which are lists of sections), why are we adding a special non-manifest dictionary item to it? And what does the message yaml file look like? I have a suspicion that `ContentBuilder` won't really apply to them.